bench-vortex/src/bin/clickbench.rs

```diff
@@ -69,12 +69,30 @@ fn main() {
         idempotent(&output_path, |output_path| {
             eprintln!("Downloading file {idx}");
             let url = format!("https://pub-3ba949c0f0354ac18db1f0f14f0a2c52.r2.dev/clickbench/parquet_many/hits_{idx}.parquet");
-            let r = reqwest::blocking::get(url)?.error_for_status()?;
-            let body = r.bytes()?;
 
-            let mut file = OpenOptions::new().create_new(true).write(true).open(output_path)?;
-            file.write_all(body.as_ref())?;
-            file.flush()?;
+
+            let make_req = || client.get(&url).send();
+            let mut output = None;
+
+            for attempt in 1..4 {
```

Comment:
Hail pervasively retries every 5XX response without bound. It is sometimes a catastrophic idea, but it makes the system robust to a lot of bad things (e.g my laptop wifi is dead for an hour while I'm commuting). Probably not the right choice for us though, heh.